bin/_blackbox_common.sh

```diff
@@ -277,6 +277,9 @@ function shred_file() {
     #NOTE: srm by default uses 35-pass Gutmann algorithm
     CMD=srm
     OPT=-f
+  elif _F=$(mktemp); rm -P "${_F}" >/dev/null 2>/dev/null ; then
```

Comment:
Sorry, I wasn't paying attention (I shouldn't type before coffee!).  You are correct, the "--" should go where the file is actually deleted.  Good catch!